src/components/PropTable.js

```diff
@@ -27,66 +116,41 @@ export default class PropTable extends React.Component {
       return null;
     }
 
-    const props = {};
-
-    if (type.propTypes) {
-      for (const property in type.propTypes) {
-        if (!type.propTypes.hasOwnProperty(property)) {
-          continue;
-        }
-        const typeInfo = type.propTypes[property];
-        const propType = PropTypesMap.get(typeInfo) || 'other';
-        const required = typeInfo.isRequired === undefined ? 'yes' : 'no';
-        props[property] = { property, propType, required };
-      }
-    }
+    const props = hasDocgen(type) ? propsFromDocgen(type) : propsFromPropTypes(type);
 
-    if (type.defaultProps) {
-      for (const property in type.defaultProps) {
-        if (!type.defaultProps.hasOwnProperty(property)) {
-          continue;
-        }
-        const value = type.defaultProps[property];
-        if (value === undefined) {
-          continue;
-        }
-        if (!props[property]) {
-          props[property] = { property };
-        }
-        props[property].defaultValue = value;
-      }
-    }
+    if (isNotEmpty(props)) {
+      const array = Object.values(props);
```

Comment:
I'd change this name to something more descriptive. `propsPropTypeInformation` maybe.